README.md

```diff
@@ -1,5 +1,9 @@
 
 [![Circle CI](https://circleci.com/gh/Financial-Times/coco-logfilter/tree/master.png?style=shield)](https://circleci.com/gh/Financial-Times/coco-logfilter/tree/master) [![Go Report Card](https://goreportcard.com/badge/github.com/Financial-Times/coco-logfilter)](https://goreportcard.com/report/github.com/Financial-Times/coco-logfilter) [![Coverage Status](https://coveralls.io/repos/github/Financial-Times/coco-logfilter/badge.svg)](https://coveralls.io/github/Financial-Times/coco-logfilter)
+
+# DEPRECATED
+This repository has been deprecated. The code was integrated into the [log-collector](https://github.com/Financial-Times/log-collector) repository
+
 # coco-logfilter
```

Comment:
I suggest deleting the rest of the readme and leaving just the deprecation notice + archive the repo